### PR TITLE
Allow mink to work with Symfony3

### PR DESCRIPTION
**Pull request by benji07**
Hey, i tried to install mink with symfony3, and it failed, so I made this PR.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,8 +11,14 @@ matrix:
       env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
       env: DEPENDENCIES=dev
+    - php: 5.6
+      env: DEPENDENCIES='dev' SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: DEPENDENCIES='dev' SYMFONY_VERSION="3.0.x-dev as 2.8"
```
Comment (stof) [inline .travis.yml]: there is no valid reason to alias this as 2.8

File: .travis.yml [hunk 2]
```diff
@@ -11,8 +11,14 @@ matrix:
       env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
       env: DEPENDENCIES=dev
+    - php: 5.6
+      env: DEPENDENCIES='dev' SYMFONY_VERSION=2.8.*@dev
```
Comment (stof) [inline .travis.yml]: we don't need both 2.8 and 3.0 here: one of them is installed by the normal ``DEPENDENCIES=dev`` version

**Comment by benji07** [comment]
@stof ready to be review / merged

**Comment by benji07** [comment]
@stof any news ?

File: .travis.yml [hunk 3]
```diff
@@ -23,6 +26,7 @@ before_install:
   - if [ "$DEPENDENCIES" = "dev" ]; then perl -pi -e 's/^}$/,"minimum-stability":"dev"}/' composer.json; fi;
 
 install:
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require --no-update "symfony/symfony:${SYMFONY_VERSION}"; fi;
```
Comment (stof) [inline .travis.yml]: given that we use a single component which itself has no dependency, no need to require the full framework.

**Comment by benji07** [comment]
@stof done

**Pull request merged**